app/models/transfer.rb

```diff
@@ -127,12 +127,14 @@ def form_comment(comment)
     def estimate_amount(out)
       estimated_amount = out ? (amount_cents + comission_cents) : amount_cents
       if currency_mismatch?
-        Money.add_rate(from_currency, to_currency, exchange_rate.to_d)
+        rate = Money.default_bank.get_rate(from_currency, to_currency)
```

Comment:
Because "Money.add_rate" set global rate for this pare currencies, i temporary save previous value to local var and then return value: Money.default_bank.add_rate(from_currency, to_currency, rate)